src/styles/utils/variables.scss

```diff
@@ -0,0 +1,12 @@
+$height-clock: 500px;
+$height-clock-center: 25px;
+$margin-clock-center: -$height-clock-center / 2;
+
+@keyframes move {
+  from {
+    transform: rotate(0);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
I'm not sure that its the right place for animation

Reply:
Better to create separate file, but it is not critical at all

Reply:
sth similar that "./utils/keyframes.scss"?